package.json

```diff
@@ -210,6 +211,8 @@
     "@stryker-mutator/karma-runner": "^3.0.1",
     "@stryker-mutator/mocha-runner": "^3.0.1",
     "@stryker-mutator/typescript": "^3.0.1",
-    "**/**/ini": "^1.3.6"
+    "**/**/ini": "^1.3.6",
+    "**/**/prismjs": "^1.23.0",
+    "**/**/serialize-javascript": "^3.1.0"
```

Comment:
What's with these additions? It is not on the other PR with the same purpose

Reply:
these are security vulnerability issue which popped up today morning

Reply:
I haven't seen this syntax before, is this pinning the version of prismjs that is a sub-dependency to top-level dependencies to a specific version?

Reply:
https://www.npmjs.com/advisories/1548
https://www.npmjs.com/advisories/1638